source/Ship.cpp

```diff
@@ -3533,37 +3533,65 @@ double Ship::TurnRate() const
 
 
 
-double Ship::Acceleration() const
+// Get maximum forward thrust. Use regular thruster and afterburner if allThrust is set.
+// Otherwise use regular thrusters if they are available and afterburner only if not.
+double Ship::ForwardThrust(bool allThrust) const
 {
 	double thrust = attributes.Get("thrust");
-	return (thrust ? thrust : attributes.Get("afterburner thrust")) / InertialMass();
+	if(allThrust)
+		return thrust + attributes.Get("afterburner thrust");
+	else
+		return thrust ? thrust : attributes.Get("afterburner thrust");
 }
 
 
 
-double Ship::MaxVelocity() const
+// Get front acceleration depending on whether regular thruster and afterburner should be combined.
+double Ship::ForwardAcceleration(bool allThrust) const
 {
-	// v * drag / mass == thrust / mass
-	// v * drag == thrust
-	// v = thrust / drag
-	double thrust = attributes.Get("thrust");
-	return (thrust ? thrust : attributes.Get("afterburner thrust")) / Drag();
+	return ForwardThrust(allThrust) / InertialMass();
 }
 
+double Ship::ReverseAcceleration() const
+{
+	return attributes.Get("reverse thrust") / InertialMass();
+}
 
+// Get overall acceleration either forward or reverse whatever is larger.
+double Ship::Acceleration(bool allThrust) const
+{
+	return max(ForwardAcceleration(allThrust), ReverseAcceleration());
+}
 
-double Ship::ReverseAcceleration() const
+// Is forward acceleration faster?
+// The result can depend on whether regular forward thruster and afterburner should be combined.
+bool Ship::PrefersForwardAcceleration(bool allThrust) const
 {
-	return attributes.Get("reverse thrust");
+	return ForwardAcceleration(allThrust) >= ReverseAcceleration();
 }
 
 
 
-double Ship::MaxReverseVelocity() const
+// Get maximum forward speed depending on whether regular thruster and afterburner should be combined.
+double Ship::MaxForwardSpeed(bool allThrust) const
+{
+	// v * drag / mass == thrust / mass
+	// v * drag == thrust
+	// v = thrust / drag
```

Comment:
```suggestion
	// v * drag / mass = thrust / mass
	// v * drag = thrust
	// v = thrust / drag
```

why boolean operations in comment XD